### PR TITLE
clarify config map <-> pod template sync in jenkins

### PR DESCRIPTION
**Pull request by gabemontero**
Fixes https://github.com/openshift/openshift-docs/issues/6664

@bparees @bmcelvee @codrinbucur ptal

### CODE REVIEW

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -578,8 +578,23 @@ After startup, the
...
-Kubernetes plug-in based on the presence or absence of the noted labels and
-annotations.
+Kubernetes plug-in.
+
+In particular, the following rules will apply:
+
+- Removal of the label or annotation from the `ConfigMap`, `ImageStream`, or
+`ImageStreamTag` will result in the deletion of any existing `PodTemplate` from
+the configuration of the Kubernetes plug-in.
+- Similarly, if those objects are in fact removed, the corresponding configuration
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: s/in fact//

File: using_images/other_images/jenkins.adoc [hunk 2]
```diff
@@ -578,8 +578,23 @@ After startup, the
...
+is removed from the Kubernetes plug-in.
+- Conversely, either the creation of appropriately labeled or annotated `ConfigMap`,
+`ImageStream`, or `ImageStreamTag` objects, or the adding of labels after their
+initial creation, leads to the creation of `PodTemplate` in the Kubernetes-plugin
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: s/creation of/creation of a/

File: using_images/other_images/jenkins.adoc [hunk 3]
```diff
@@ -578,8 +578,23 @@ After startup, the
...
+configuration.
+- In the case of the `PodTemplate` via `ConfigMap` form, changes to the `ConfigMap`
+data for the `PodTemplate`will be applied to the `PodTemplate` settings in the
+Kubernetes plug-in configuration, and will even override any changes made to the
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: s/even//

**Review by bparees — CHANGES_REQUESTED** [review]

**Comment by gabemontero** [comment]
@bparees comment updates pushed

**Comment by bparees** [comment]
lgtm.

**Comment by bmcelvee** [comment]
LGTM. Is this ready to merge? If so, I'll merge and make some minor formatting edits in a follow-up PR. Thanks!

**Comment by gabemontero** [comment]
yep this is for existing / already release functino @bmcelvee - go ahead and merge at your discretion

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../using_images/other_images/jenkins.adoc#using-images-other-images-jenkins[Jenkins]
|Clarified ConfigMap and PodTemplate sync definitions in xref:../using_images/other_images/jenkins.adoc#using-the-jenkins-kubernetes-plug-in-to-run-jobs[Using the Jenkins Kubernetes Plug-in to Run Jobs] section.
%

**Pull request merged**